tests/UI/campaigns/functional/BO/11_international/02_locations/03_states/03_bulkActionsStates.js

```diff
@@ -26,8 +26,8 @@ let page;
 let numberOfStates = 0;
 
 const statesToCreate = [
-  new StateFaker({name: 'todelete1'}),
-  new StateFaker({name: 'todelete2'}),
+  new StateFaker({name: 'todelete1', isoCode: 'HM'}),
+  new StateFaker({name: 'todelete2', isoCode: 'BV'}),
```

Comment:
Why did you hardcode these values when faker is taking random values from the list you provided ?

Reply:
The array `statesIsoCodes` is used in CRUDState test : there will be no problem because we create 1 state and we update it
=> So even if State 1 iso code is equal to State 2 iso code, there will be no problem.

For the bulk actions test, we create 2 state at the same time.
=> So using the array, there is `1/64` chance that State 1 iso code is equal to State 2 iso code